### PR TITLE
A blog with Webiny and Eleventy

### PR DESCRIPTION
**Pull request by kmaurice012**

### CODE REVIEW

**Comment by kmaurice012** [comment]
A blog with Webiny and Eleventy

File: tutorials/EleventyWebinyBlog/_includes/layouts/post.njk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <div class="container">
+    <div class="columns">
+      <div class="column is-8 is-offset-2">
+        <ul>
+          {%- for post in post.posts -%}
+            <article class="content mt-5">
+              <h1 class="title">{{ post.title }}</h1>
+              <p class="subtitle is-6">
+                Author: {{post.author.name}}
+                {# <time datetime="{{ post.date }}">{{ post.date | readableDate }}</time> #}
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_includes/layouts/post.njk:15]: Why is this commented out? Should it be removed?

File: tutorials/EleventyWebinyBlog/_data/site.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "My Eleventy Blog",
+  "url": "https://example.com/",
+  "language": "en-US",
+  "description": "A simple blog with awesome content"
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_data/site.json:5]:
```suggestion
  "title": "Source Webiny Headless CMS",
  "url": "https://example.com/",
  "language": "en-US",
  "description": "An Eleventy blog template for sourcing content from Webiny Headless CMS"
```

File: tutorials/EleventyWebinyBlog/_site/about/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          <h1>About Me</h1>
+          <p>I am a person that writes stuff.</p>
+
+        </div>
+      </div>
+    </div>
+  </div>
+</article>
+
+  </main>
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_site/about/index.html:45]: Please format the code, there are a lot of gaps and irregularities, here for example.

**Review by endymion1818 — CHANGES_REQUESTED** [review]

**Comment by endymion1818** [comment]
Hi @kmaurice012, a few things to tidy up but nothing major. Thanks for the PR!

**Comment by kmaurice012** [comment]
Done

File: tutorials/EleventyWebinyBlog/_site/indeex/index.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<h1>Hello, world!</h1>
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_site/indeex/index.html:1]: Is this file spurious?

File: tutorials/EleventyWebinyBlog/_site/index.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    <a href="/about" class="navbar-item">
+      About Me
+    </a>
+  </div>
+</nav>
+</header>
+
+  <main>
+    
+<section class="hero is-medium is-primary is-bold">
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_site/index.html:31]: formatting

File: tutorials/EleventyWebinyBlog/_site/index.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+            </footer>
+          </div>
+        </div>
+      
+    </div>
+  </div>
+</section>
+  </main>
+</body>
+</html>
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_site/index.html:121]: formatting

File: tutorials/EleventyWebinyBlog/_site/posts/first-post/index.html [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+    <a href="/about" class="navbar-item">
+      About Me
+    </a>
+  </div>
+</nav>
+</header>
+
+  <main>
+
```
Comment (endymion1818) [inline tutorials/EleventyWebinyBlog/_site/posts/first-post/index.html:30]: formatting

**Review by endymion1818 — COMMENTED** [review]
Nearly there!

**Comment by kmaurice012** [comment]
Done

**Review by endymion1818 — APPROVED** [review]

**Pull request merged**